assets/common/components/mlflow_model_local_validation/spec.yaml

```diff
@@ -1,15 +1,15 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 
 name: mlflow_model_local_validation
-version: 0.0.3
+version: 0.0.4
 type: command
 
 is_deterministic: True
 
 display_name: MLFlow model local validation
 description: Validates if a MLFLow model can be loaded on a compute and is usable for inferencing.
 
-environment: azureml://registries/azureml/environments/python-sdk-v2/versions/6
+environment: azureml://registries/azureml/environments/python-sdk-v2/versions/7
```

Comment:
Should we update all other components that has version 6 in the env?

Reply:
all 5 components are updated